tests/helpers.py

```diff
@@ -286,12 +287,13 @@ def start_controller(self, clean_config=False, enable_all=False,
             #    pid = int(f.read())
             #    os.kill(pid, signal.SIGTERM)
             time.sleep(5)
-            if os.path.exists('/var/run/kytos/kytosd.pid'):
+            pid_path = os.path.join(BASE_ENV, 'var/run/kytos/kytosd.pid')
```

Comment:
@HeriLFIU, if you could also parametrize a `extra_args=os.environ.get("KYTOSD_EXTRA_ARGS", "")` on start_controller, that'd be great. It turns out that when running test locally, maybe developers also want to parametrize or pass extra args to `kytosd`, for instance, I typically use a different `-n /home/path/napps`:

```
+++ b/tests/helpers.py
@@ -278,7 +278,8 @@ class NetworkTest:
         self.db_client.drop_database(self.db_name)
 
     def start_controller(self, clean_config=False, enable_all=False,
-                         del_flows=False, port=None, database='mongodb'):
+                         del_flows=False, port=None, database='mongodb',
+                         extra_args=os.environ.get("KYTOSD_EXTRA_ARGS", "")):
         # Restart kytos and check if the napp is still disabled
```

if `extra_args` is different than `""`, then you can append it to the `daemon` string with an extra space upfront the value, so if a user passes `"-n whatever -c foo"` the expected result is `" -n whatever -c foo"`, just so it can get appended nicely without developers having to be concerned about the initial empty space char to append the string. See what I mean?

Reply:
I appended the extra arguments at the end of the daemon string. Adding additional arguments with the "EXTRA_ARGS" environment variable should work now.